### DISCUSSION TITLE
Fine granually configure access for Secret Service

### DISCUSSION THREAD

**Discussion by frere-jacques** [keepassxreboot/keepassxc#9010] [Ideas]
I set up keepassxc as gnome-keyring replacement and currently evolution is using keepassxc to store the passwords.  
In Secret Service configuration I kept all checkmarks as default, because they appear reasonable. But that means I have to allow access to keepassxc for evolution all the time, which is a bit annoying.

While I don't want a program called crazy-password-grabber to access the evolution password I would like to let evolution to grab the password without further notice. Would it be possible to make it configurable that way?

**Comment by michaelk83** [comment]
Not yet: https://github.com/keepassxreboot/keepassxc/issues/7571#issuecomment-1076921523

**Reply by frere-jacques** [reply]
Thanks for reply.
